reflect-ballerina/build.gradle

```diff
@@ -90,7 +90,7 @@ task copyStdlibs(type: Copy) {
     }
 }
 
-task updateTomlFile {
+task updateTomlVerions {
```

Comment:
```suggestion
task updateTomlVersions {
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-reflect/pull/51/commits/b5bc7adf0c1cba24ef1b3f09c6ca065d73766911